test/sslapitest.c

```diff
@@ -1911,6 +1911,9 @@ static int test_tlsext_status_type(void)
     /* First just do various checks getting and setting tlsext_status_type */
 
     clientssl = SSL_new(cctx);
+    if (!TEST_ptr(clientssl))
+        goto end;
+        
```

Comment:
nit: you've got a trailing whitespace character here

Reply:
That can be removed when merging.

Reply:
> nit: you've got a trailing whitespace character here

I'll fix it